ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -120,13 +120,6 @@ class TestCommon {
          */
         fun parseErrorResponse(response: Response): IdpResponseDecorator {
             return if (response.header("Location") != null) {
-                /*
-                 * TODO "Manually change DDF IdP to respond with 302/303 status code for Redirect"
```

Comment:
❓ This TODO was to remind us that when DDF is fixed to return the correct HTTP status code. What was the reasoning behind taking it out?

Reply:
I removed it cause we aren't doing that anymore. We will be responding exclusively with POST. We still might want to do it for log out so I will document that in the ticket.